### DISCUSSION TITLE
<empty> OperationName in Application Insights

### DISCUSSION THREAD

**Discussion by scleynen** [arcus-azure/arcus.observability#218] [Q&A] [answered]
I've noticed that in Application Insights the OperationName is empty when using the Arcus WebApi libraries. Is there a way to putt here a value?

![image](https://user-images.githubusercontent.com/7026342/121317869-fb6cce80-c90a-11eb-851a-db7e128f1f1c.png)

**Comment by tomkerkhove** [comment, marked as answer]
This will be fixed in https://github.com/arcus-azure/arcus.observability/issues/165

**Comment by pim-simons** [comment]
See also https://github.com/arcus-azure/arcus.observability/pull/229
